src/Microsoft.Health.Fhir.Web/DevelopmentIdentityProviderConfiguration.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.Health.Fhir.Web
     public class DevelopmentIdentityProviderConfiguration
     {
         public const string Audience = "fhir-api";
+        public const string LastModifiedClaim = "appid";
```

Comment:
Is there any particular reason for changing this from `claim_id` to `appid`?

Reply:
I was originally looking for a claim that was in both AAD and IdentityServer tokens, so I switched client_id to appid, but now it doesn't matter anymore.  Since I already changed the integration tests to look for appid, let's keep this as appid.  I will also revert the change to StartupWithTraceAuditLogger that I made earlier, since we no longer need it.